src/components/character-count/index.md.njk

```diff
@@ -104,6 +104,49 @@ For example, ‘Enter a summary’.
 Say ‘[whatever it is] must be [number] characters or less’.<br>
 For example, ‘Summary must be 400 characters or less’.
 
+## Use localisation to translate ‘character count’ into other languages
+
+Read how to [set up localisation with Nunjucks and Javascript](/get-started/localisation/).
```

Comment:
Change sentence to remove 'set up'. Maybe to 'Read how to use localisation with...'